internal/pkg/cli/env_delete.go

```diff
@@ -173,12 +173,10 @@ func (o *deleteEnvOpts) Execute() error {
 	}
 	o.prog.Stop(log.Ssuccessf(fmtRetainEnvRolesComplete, o.name))
 
-	o.prog.Start(fmt.Sprintf("Deleting resources for the %q environment\n", o.name))
+	// DeleteStack streams the deletion events; we don't need a spinner over top of it.
```

Comment:
Why do we not need the spinner here? Is it because we don't have the spinner in `svc delete`?

Reply:
Putting the spinner in here causes it to constantly overwrite the last line of the stack streamer and blink/output endless useless lines in the terminal. Removing the start call here fixes that.